amp/src/test/java/org/digijava/kernel/ampapi/endpoints/activity/validators/ValueValidatorTest.java

```diff
@@ -117,4 +127,19 @@ public void testPossibleValueFYInvalid() throws Exception {
                 valueValidator.isValid(importer, newFieldParent, fyFieldDescription, FY_FIELD));
         assertEquals(ActivityErrors.FIELD_INVALID_VALUE, valueValidator.getErrorMessage());
     }
+
+    @Test
+    public void testDraftSubmissionDraftFMDisabled() throws Exception {
```

Comment:
The exception is never thrown. I think we need to remove all the Exception throws (including above tests)

Reply:
Oh, you are right, I took an existing test case from above as a template and didn't notice this, let me remove them.